ngraph/test/frontend/onnx/telemetry.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "telemetry.hpp"
+
+#include "onnx_utils.hpp"
+
+using namespace ngraph;
+using namespace ov::frontend;
+
+using ONNXTelemetryTest = FrontEndTelemetryTest;
+
+static TelemetryFEParam getTestData_relu() {
+    TelemetryFEParam res;
+    res.m_frontEndName = ONNX_FE;
+    res.m_modelsPath = std::string(TEST_ONNX_MODELS_DIRNAME);
+    res.m_modelName = "external_data/external_data.onnx";
```

Comment:
Can we use a model with a subgraph in this test? For example with Loop operation. As it is important to collect "op_count" in inner subgraphs. I can provide such model if needed.
Same question about TF model.

Reply:
It would be great if you could provide a such model for ONNX.
about Paddle and TF: currently we do not support Loop/TF or other subgraph operation via these frameworks

Reply:
done